### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by alexonab**

### CODE REVIEW

**Comment by tedchou12** [comment]
Thank you for fixing this so quickly!!

**Comment by tradinginaction** [comment]
thank you too 💯

File: webull/webull.py [hunk 1]
```diff
@@ -744,13 +751,14 @@ def get_news(self, stock=None, Id=0, items=20):
         params = {'currentNewsId': Id, 'pageSize': items}
         return requests.get(self._urls.news(self.get_ticker(stock)), params=params).json()
 
-    def get_bars(self, stock=None, tId = None, interval='m1', count=1, extendTrading=0) :
+    def get_bars(self, stock=None, tId = None, interval='m1', count=1, extendTrading=0, timeStamp=None):
```
Comment (imvinaypatil) [inline webull/webull.py:754]: would be better if kept as `timestamp` with small s

**Comment by imvinaypatil** [comment]
hi @alexonab, Is there any api available to pull the watchlists. I tried to inspects it via browser but couldn't find out the endpoint. It would be very helpful if you can help.
Thanks.

**Comment by dougmbaya** [comment]
> hi @alexonab, Is there any api available to pull the watchlists. I tried to inspects it via browser but couldn't find out the endpoint. It would be very helpful if you can help.
> Thanks.

+1 Yes and also add and remove (edit the watch list)

**Comment by imvinaypatil** [comment]
Never mind I fund the api to get the watchlists. I'll try to get this implemented. Thanks

**Comment by dougmbaya** [comment]
@imvinaypatil What is the API? Does it also allow you to get your screens?

**Comment by imvinaypatil** [comment]
@dougmbaya 
this is the endpoint to pull watchlists `https://quotes-gw.webullbroker.com/api/personal/portfolio/v2/check` but I'm facing a Invalid Token exception on calling this endpoint from the Webull class. 

I don't know what you meant by screens btw

**Comment by dougmbaya** [comment]
Thank you @imvinaypatil 
Webull has a stock screening section where you research stocks based on certain criteria. See image below.
image

**Comment by tarek-vader** [comment]
> @dougmbaya
> this is the endpoint to pull watchlists `https://quotes-gw.webullbroker.com/api/personal/portfolio/v2/check` but I'm facing a Invalid Token exception on calling this endpoint from the Webull class.
> 
> I don't know what you meant by screens btw

am also interested, did you find a solution to pull watchlist tickers?

**Pull request merged**